### DISCUSSION TITLE
i want give you a traduction (idiomas, querida.)

### DISCUSSION THREAD

**Discussion by MascarellRikTk** [community/community#123334] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hola Cesar. Estoy actualmente estudiando esperanto y dado que tu emulador tiene soporte para diferentes idiomas me gustaría hacerte la traducción a ese idioma. Yo practico, tu tienes un paquete de idioma gratis y la comunidad esperantista tiene un emulador de cabecera, todos ganamos.
Piénsalo y me dices

**Comment by ebndev** [comment]
Hi @MascarellRikTk, thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
